app/models/project_manager.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectManager < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :tasks, dependent: :destroy
```

Comment:
This probably wouldn't be applicable in the real world (they'd probably get handed off to another project manager), but no other logic was specified so without their PM they'll just get removed